src/components/Tool/Appeal/AppealProgressBar.tsx

```diff
@@ -114,8 +129,8 @@ const AppealProgressBar = ({
                 backgroundColor: theme.palette.progressBarYellow.main,
```

Comment:
Could we make these progress bars use the same styling as the progress bar on the Dashboard?

Reply:
These are the colours we use for all process bars throughout the site. The dashboard is different as it prob needed to stand out and that colour worked better for the design.

`progressBarYellow`
`progressBarOrange`
`progressBarGray`